doc/rose-configuration.html

```diff
@@ -1239,6 +1239,16 @@ <h2 id="appendix-rose-ana-config">Appendix: rose-ana
                 <code>rose task-run</code>
               </li>
             </ul>
+
+            <p>The app may also define a configuration section; 
+              <code>[ana:config]</code>, whose key-value pairs define app-wide
+              settings that are passed through to the analysis classes.  In 
```

Comment:
Double spaces achieve nothing in html. I think in the past they could cause problems with strange symbols appearing but it seems to render in Firefox fine.